src/reactions/aggregateReactions.js

```diff
@@ -23,7 +23,7 @@ export default (reactions: EventReaction[], ownEmail: string): AggregatedReactio
           }
         }
 
-        if (x.user && x.user.email === ownEmail) {
+        if (ownEmail && x.user && x.user.email && x.user.email === ownEmail) {
```

Comment:
Can `ownEmail` be undefined?

Reply:
May be if we don't pass to the function.

Actually I was writing tests, and found this :)

Reply:
I would prefer to figure out why `ownEmail` is undefined and fix this separately. 
It might be causing other issues too.
Then remove this commit.

Reply:
Actually by 'I would prefer' I meant 'the correct way to approach this' :)

Reply:
Actually as soon as I added `user: { user_id: 2 },` in reaction object in `every duplicate reaction is aggregated to a single one with appropriate count` it started failing. Because now `x.user` is present but, `x.user.email` is null and also we aren't passing `ownEmail` in this test, it started failing.